AndroidCompat/getAndroid.sh

```diff
@@ -20,15 +20,15 @@ fi
 
 
 # foolproof against running from AndroidCompat dir instead of running from project root
-if [ "$(basename $(pwd))" = "AndroidCompat" ]; then
+if [ "$(basename "$(pwd)")" = "AndroidCompat" ]; then
```

Comment:
why not `\"` ?

Reply:
it will fail if there's space in pwd, good find

Reply:
\" in that place would actually make the string end